packages/ts-client/src/index.ts

```diff
@@ -18,6 +18,8 @@ export type { AuthErrorReason } from './auth.js';
 
 export { isAuthError, AUTH_ERROR_REASONS } from './auth.js';
 
+export { sdkVersion } from './sdkVersion';
```

Comment:
Why this line was added?

Reply:
I recreated the protos submodule. There is no tags and I forgot to pull a 0.9.0 compatible commit.
Will undo.